core/src/main/java/com/amplifyframework/devmenu/PersistentLogStoragePlugin.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.devmenu;
```

Comment:
Don't know if it's worth changing now but was it discussed whether this would make more sense in the logging package vs. under dev menu?

Reply:
In the PR to store logs for the developer menu, Jameson left a comment about this: "One could argue either way. I'd say devmenu, since its the only consumer we currently expect for the plugin. If we find another use for it, we can always move it back to the more general location, later."